### ISSUE TITLE
Screen readers read data keys instead of the date in `time-comparison` bar charts 

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/wc-admin#1543]
- Go to the _Products_ report.
- Change to a bar chart.
- Use a screen reader to navigate the bars.
- For the first bar of the screenshot below, something like `Quarter to Date (Jan 1 - Feb 13, 2019) 572` is read, while it should be `Week of January 1, 2019 572`.

![image](https://user-images.githubusercontent.com/3616980/52709530-4cfc2d00-2f8d-11e9-8c4e-a6ff9e570b91.png)
